### PR TITLE
Remove hub related code (DAT-13866)

### PR DESCRIPTION
**Pull request by abrackx**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [ ] Enhancement/New feature (adds functionality without impacting existing logic)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
 
## Description
‼️ This is a breaking change ‼️ 
Removes all hub functionality, including hub related commands and hub operation reporting.

### CODE REVIEW

File: liquibase-core/src/main/java/liquibase/command/core/ChangelogSyncCommandStep.java [hunk 1]
```diff
@@ -56,32 +49,15 @@ public List<Class<?>> requiredDependencies() {
...
         final ChangeLogParameters changeLogParameters = (ChangeLogParameters) commandScope.getDependency(ChangeLogParameters.class);
 
-        BufferedLogService bufferLog = new BufferedLogService();
-        HubHandler hubHandler = null;
-
-        try {
-            ChangeLogIterator runChangeLogIterator = buildChangeLogIterator(tag, changeLog, changeLogParameters.getContexts(), changeLogParameters.getLabels(), database);
-            CompositeLogService compositeLogService = new CompositeLogService(true, bufferLog);
+        ChangeLogIterator runChangeLogIterator = buildChangeLogIterator(tag, changeLog, changeLogParameters.getContexts(), changeLogParameters.getLabels(), database);
+        CompositeLogService compositeLogService = new CompositeLogService(true);
```
Comment (StevenMassaro) [inline liquibase-core/src/main/java/liquibase/command/core/ChangelogSyncCommandStep.java:56]: Maybe this log service and scope.child on 58 is not needed.

**Review by StevenMassaro — APPROVED** [review]

File: liquibase-core/src/main/java/liquibase/Liquibase.java [hunk 2]
```diff
@@ -746,26 +624,24 @@ public void run() throws Exception {
...
      * logIterator.
      */
-    private void doRollback(BufferedLogService bufferLog, String rollbackScript, ChangeLogIterator logIterator, Contexts contexts, LabelExpression labelExpression, HubUpdater hubUpdater, Operation rollbackOperation) throws Exception {
-        CompositeLogService compositeLogService = new CompositeLogService(true, bufferLog);
+    private void doRollback(String rollbackScript, ChangeLogIterator logIterator, Contexts contexts, LabelExpression labelExpression) throws Exception {
+        LogService logService = Scope.getCurrentScope().get(Scope.Attr.logService, LogService.class);
         if (rollbackScript == null) {
             List<ChangesetsRolledback.ChangeSet> processedChangesets = new ArrayList<>();
-            Scope.child(Scope.Attr.logService.name(), compositeLogService, () -> {
+            Scope.child(Scope.Attr.logService.name(), logService, () -> {
```
[the remark this answers is not in the document]
Comment (abrackx) [reply liquibase-core/src/main/java/liquibase/Liquibase.java:631]: Good catch, I think I've resolved all of these instances.

File: liquibase-core/src/main/java/liquibase/logging/core/BufferedLogger.java [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 import java.util.logging.Level;
 
+@Deprecated
 public class BufferedLogger extends AbstractLogger {
 
     private BufferedLogService bufferedLogService;
```
Comment (wwillard7800) [inline liquibase-core/src/main/java/liquibase/logging/core/BufferedLogger.java:10]: We still use the `BufferedLogger` in the Pro CommandTests.  Should we just move that code to the Pro tests world?

**Review by wwillard7800 — APPROVED** [review]
Looks good one question/comment

File: liquibase-core/src/main/java/liquibase/logging/core/BufferedLogger.java (hunk 3 above)
Comment (abrackx) [reply liquibase-core/src/main/java/liquibase/logging/core/BufferedLogger.java:10]: I was talking about this with @filipelautert, we also use it in [here](../blob/master/liquibase-cdi/src/test/java/liquibase/integration/cdi/CDILiquibaseTest.java#L31) in core. I created a [task](https://datical.atlassian.net/browse/DAT-14007) to figure out what to do with it.

**Pull request merged**